book/book.toml

```diff
@@ -10,10 +10,10 @@ site-url = "/noname/"
 use-site-url-as-root = true
 curly-quotes = true
 git-repository-url = "https://www.github.com/mimoo/noname"
-additional-css = ["././mdbook-admonish.css"]
+additional-css = ["./mdbook-admonish.css"]
```

Comment:
this was a change I did manually aside from running the command found in the failed job log - we previously seemed to be using a wrong path?

Reply:
Yeah, it looks like a wrong path.